lib/katello_deploy/box_loader.rb

```diff
@@ -62,20 +69,13 @@ def process_versions(config, versions)
           Marshal.dump(config['boxes']['centos7-foreman-nightly'])
         )
         config['boxes']["centos7-foreman-#{version}"]['options'] << " --version #{version}"
-      end
 
-      versions['katello'].each do |version|
-        config['boxes']["centos6-katello-#{version}"] = Marshal.load(
-          Marshal.dump(config['boxes']['centos6-katello-nightly'])
-        )
-        config['boxes']["centos6-katello-#{version}"]['options'] << " --katello-version #{version}"
-      end
+        next unless (katello_version = versions['mapping'][version])
 
-      versions['katello'].each do |version|
-        config['boxes']["centos7-katello-#{version}"] = Marshal.load(
+        config['boxes']["centos7-katello-#{katello_version}"] = Marshal.load(
           Marshal.dump(config['boxes']['centos7-katello-nightly'])
         )
-        config['boxes']["centos7-katello-#{version}"]['options'] << " --katello-version #{version}"
+        config['boxes']["centos7-katello-#{katello_version}"]['options'] << " --version #{version}"
```

Comment:
Sure, we changed over to be Foreman centric, and thus derive Katello versions and boxes off of that. However, since the scenario changes are in 3.0+ the version setup did not need to be updated yet. So what this is doing is, creating a Katello box for each entry here -- https://github.com/Katello/katello-deploy/pull/210/files#diff-66fb1e4b3ef045958fd3fe6e6dfe1623R9